doc/website/release-notes/iceoryx-unreleased.md

```diff
@@ -362,6 +363,25 @@
     #include "iceoryx_hoofs/cxx/algorithm.hpp"
     constexpr uint32_t MAX_VAL = algorithm::maxVal(3, 1890, 57);
     constexpr uint32_t MIN_VAL = algorithm::minVal(3, 1890, 57);
+
+20. `ReleativePointer::registerPtr` returns `cxx::optional`
+
+    ```cpp
+    // before
+    uint64_t id = RelativePointer::register(myPtr, 42);
+
+    if(id == INVALID_ID)
```

Comment:
Note that normally that should never happen if there is sufficient space in the repository. If we see that the current setting is too limiting we need to increase it, but it is optimized for thousands of ids but not millions (brute force search during registration, but we could hash later if needed).